Assets/_BForBoss/_Core/Scripts/EnergySystem/Data/EnergySO.cs

```diff
@@ -0,0 +1,25 @@
+using UnityEngine;
+
+namespace BForBoss
+{
+    [CreateAssetMenu(fileName = "Energy Data", menuName = "PerigonGames/EnergySystem/EnergyData", order = 1)]
+    public class EnergySO : ScriptableObject
+    {
+        [SerializeField, Tooltip("Player's current value of Energy"), Min(0f)] 
```

Comment:
Not sure if these tooltips are accurate, let me know if I should edit them

Reply:
Yes that is accurate. hmm, I think it's better to call this `_startingValue`. 

But leave the `Energy.Value` as just value.